keyboards/ogurec/readme.md

```diff
@@ -0,0 +1,16 @@
+# ogurec
+
+![ogurec](https://imgur.com/gallery/OFF1Vyp)
+
+ogurec is a reversible 12u x 3u ortholinear keyboard that offers left-sided and right-sided PM placement
+
+* Keyboard Maintainer: [H. Bond](https://github.com/drhigsby)
+* Hardware Supported: ogurec (https://github.com/drhigsby/ogurec)
+* Hardware Availability: open source 3DP case available here https://github.com/drhigsby (case creator: https://github.com/dpalka15)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?

Reply:
I added instructions: https://github.com/qmk/qmk_firmware/pull/13242/commits/a607beb364466df11556c6a252ee52b087ba996a